data/avgi/avgi side missions.txt

```diff
@@ -0,0 +1,1067 @@
+				`	"Don't the Aberrant still have superior technology?"`
+					goto technology
+				`	"And now you're able to hold your own against them?"`
+
+			`	"Discomfort. Maybe at first, when we could count on having a numerical advantage, but the Aberrant continue to reproduce at a faster rate than we can build ships. And these new weapons, well, they were considered obsolete for a reason. We need something that can dish out more damage, and at a longer range, without being as expensive and resource-intensive as a missile."`
+				goto shields
+
+			label technology
+				`	"Oh, yes, appreciation. But they don't use it to its full extent, probably because they aren't intelligent beings in their own right. They just make more of each other, without any changes other than occasional mistakes. And their weapons don't seem like they were meant to be weapons, if that makes sense. More of a cutting beam than a true weaponized laser."`
+
+			label shields
+			`	"That's why things really were desperate, until you humans came around. That changed everything, knowing there was a non-hostile, if not outright friendly, alien power nearby. Even though it sounds like your government isn't able to help directly, given the lack of connecting starways, technological support from a more advanced species could be enough for us to win. Shields, shared by your people, were already a game changer by themselves."`
+
+			choice
+				`	"I'm not sure humanity would be able to help all that much, with our technology."`
+					goto help
+				`	"Wait, you got shields from us?"`
+			`	"From the humans that were stranded here, yes, gratitude. They let us dismantle their ship's shield generator and explained the rough concept to us, and we've been able to produce our own imitation, though I hear it isn't nearly as good."`
+			
+			label help
+			`	"Dismissal. I've seen the <ship>, and the ship your friends arrived on. That's the kind of firepower we need to take on the Aberrant and win."`
+			choice
+				`	"If you say so."`
+				`	"I hope you're right."`
+			`	With that, you turn to lighter topics, as Claret finishes her drink. You learn about the new traditions the Twilight Guard is trying to promote to ensure unity and cohesion within its ranks, trying to bind together former members of dozens of independent militaries into a united whole. This apparently involves a lot of travel to promote inter-unit bonding, as well as frequent cycling between the front lines and the Outer Limits.`
+			`	Eventually, Claret is ready to leave. "Anyway, I'll see you around. And, a word of advice. If you're ever on a Twilight Guard station, look out for officers or Guardsmen who are looking for a ride. We often have people trying to make an impromptu trip, and our logistics have been so overstretched lately they're willing to contract with outside captains, if they're trustworthy."`
+			`	"I'll keep that in mind," you say.`
+			decline
+			action
+				set "avgi: Twilight Guard Transport Jobs"
```

Comment:
These don't fire. (the decline shows up as part of the conversation text.) Don't know why.

Reply:
Because of capitalization.
```suggestion
			decline
			action
				set "Avgi: Twilight Guard Transport Jobs"
```

Reply:
Thanks for catching that, though shouldn't the incorrect variable still get set here? Or is that not how it works?
(For context when I tested this the mission didn't decline until I removed one tab from the decline line, and the variable isn't present in my save no matter what I do.)

Reply:
Hmm, good point.
Maybe this then?

```suggestion
			action
				set "Avgi: Twilight Guard Transport Jobs"
			decline
```

Reply:
Yes, that works. Though I had to remove one tab from the decline to stop having it show up in the text. Dunno if that might cause other problems though.